drake/systems/plants/collision/test/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+if (fcl_FOUND)
+  set(Boost_DEBUG TRUE)
+  find_package(Boost REQUIRED system)
```

Comment:
can't accept with this in here as is.  it will blow up the matlab code (on platforms that may not be covered by our CI).